### PR TITLE
Settings fixes

### PR DESCRIPTION
**Pull request by gabriel**
- Using state for email, passphrase, invite inputs (this also fixes resetting when coming back and slowness and general weirdness)
- Remove update payment tab
- Fix invites
- Invite message box stays visible once shown
- Invite generated shows email or link confirm message correctly
- Move banner into common-adapters
- Fix invite error handling

PendingEmailInvite and PendingURLInvite were overkill. All pending invites have links, and the email pending invite includes an email. Nuked those, and if email is present, then it is a pending invite with an email. No need to get fancy with types in this case.

See https://keybase.atlassian.net/browse/DESKTOP-2271

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

**Comment by keybase-ci-visdiff** [comment]
The commits 142897a31304...0546649be3ad introduce visual changes on linux. <details><summary>:mag_right: 9 changed</summary>

 * Changed: **Update_Passphrase-Normal_Show_Typing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Normal_Show_Typing-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Normal_Show_Typing-0546649be3ad.png)  
   Update_Passphrase-Normal_Show_Typing comparison
 * Changed: **Update_Passphrase-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Normal-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Normal-0546649be3ad.png)  
   Update_Passphrase-Normal comparison
 * Changed: **Update_Passphrase-Error_Wrong_Passphrase** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Error_Wrong_Passphrase-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Error_Wrong_Passphrase-0546649be3ad.png)  
   Update_Passphrase-Error_Wrong_Passphrase comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Requirements** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Error_New_Passphrase_Requirements-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Error_New_Passphrase_Requirements-0546649be3ad.png)  
   Update_Passphrase-Error_New_Passphrase_Requirements comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Mismatch** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-0546649be3ad.png)  
   Update_Passphrase-Error_New_Passphrase_Mismatch comparison
 * Changed: **Settings_Nav-Normal_Good_Banner** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Settings_Nav-Normal_Good_Banner-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Settings_Nav-Normal_Good_Banner-0546649be3ad.png)  
   Settings_Nav-Normal_Good_Banner comparison
 * Changed: **Settings_Nav-Normal_Bad_Banner** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Settings_Nav-Normal_Bad_Banner-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Settings_Nav-Normal_Bad_Banner-0546649be3ad.png)  
   Settings_Nav-Normal_Bad_Banner comparison
 * Changed: **Settings_Nav-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Settings_Nav-Normal-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Settings_Nav-Normal-0546649be3ad.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Settings_Nav-Normal.png)
 * Changed: **Invites-Normal_Email_Error** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Invites-Normal_Email_Error-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Invites-Normal_Email_Error-0546649be3ad.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481847691929-142897a31304-0546649be3ad-linux/Invites-Normal_Email_Error.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 142897a31304...00e95ef225d5 introduce visual changes on linux. <details><summary>:mag_right: 9 changed</summary>

 * Changed: **Update_Passphrase-Normal_Show_Typing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Normal_Show_Typing-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Normal_Show_Typing-00e95ef225d5.png)  
   Update_Passphrase-Normal_Show_Typing comparison
 * Changed: **Update_Passphrase-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Normal-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Normal-00e95ef225d5.png)  
   Update_Passphrase-Normal comparison
 * Changed: **Update_Passphrase-Error_Wrong_Passphrase** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Error_Wrong_Passphrase-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Error_Wrong_Passphrase-00e95ef225d5.png)  
   Update_Passphrase-Error_Wrong_Passphrase comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Requirements** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Error_New_Passphrase_Requirements-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Error_New_Passphrase_Requirements-00e95ef225d5.png)  
   Update_Passphrase-Error_New_Passphrase_Requirements comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Mismatch** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-00e95ef225d5.png)  
   Update_Passphrase-Error_New_Passphrase_Mismatch comparison
 * Changed: **Settings_Nav-Normal_Good_Banner** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Settings_Nav-Normal_Good_Banner-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Settings_Nav-Normal_Good_Banner-00e95ef225d5.png)  
   Settings_Nav-Normal_Good_Banner comparison
 * Changed: **Settings_Nav-Normal_Bad_Banner** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Settings_Nav-Normal_Bad_Banner-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Settings_Nav-Normal_Bad_Banner-00e95ef225d5.png)  
   Settings_Nav-Normal_Bad_Banner comparison
 * Changed: **Settings_Nav-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Settings_Nav-Normal-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Settings_Nav-Normal-00e95ef225d5.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Settings_Nav-Normal.png)
 * Changed: **Invites-Normal_Email_Error** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Invites-Normal_Email_Error-142897a31304.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Invites-Normal_Email_Error-00e95ef225d5.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481849087296-142897a31304-00e95ef225d5-linux/Invites-Normal_Email_Error.png)
</summary>

**Comment by chromakode** [comment]
As a general note, we should be persisting values from forms in `routeState` so that the field values aren't lost when the user switches tabs. @chrisnojima and I have discussed this a bit and he's worked on it a bit in his recompose branch. We shouldn't be setting the `routeState` on every update; debouncing or saving on unmount is the way to go. However removing `routeState` entirely opens up another class of UX issues -- it's really frustrating to lose text you were typing on.

**Comment by keybase-ci-visdiff** [comment]
The commits 33b46869301b...6eb4211daf5a introduce visual changes on linux. <details><summary>:mag_right: 2 removed, 7 changed</summary>

 * Changed: **Update_Passphrase-Normal_Show_Typing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Normal_Show_Typing-33b46869301b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Normal_Show_Typing-6eb4211daf5a.png)  
   Update_Passphrase-Normal_Show_Typing comparison
 * Changed: **Update_Passphrase-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Normal-33b46869301b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Normal-6eb4211daf5a.png)  
   Update_Passphrase-Normal comparison
 * Changed: **Update_Passphrase-Error_Wrong_Passphrase** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Error_Wrong_Passphrase-33b46869301b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Error_Wrong_Passphrase-6eb4211daf5a.png)  
   Update_Passphrase-Error_Wrong_Passphrase comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Requirements** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Error_New_Passphrase_Requirements-33b46869301b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Error_New_Passphrase_Requirements-6eb4211daf5a.png)  
   Update_Passphrase-Error_New_Passphrase_Requirements comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Mismatch** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-33b46869301b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-6eb4211daf5a.png)  
   Update_Passphrase-Error_New_Passphrase_Mismatch comparison
 * Changed: **Settings_Nav-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Settings_Nav-Normal-33b46869301b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Settings_Nav-Normal-6eb4211daf5a.png)  
   Settings_Nav-Normal comparison
 * Changed: **Invites-Normal_Email_Error** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Invites-Normal_Email_Error-33b46869301b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Invites-Normal_Email_Error-6eb4211daf5a.png)  
   Invites-Normal_Email_Error comparison
 * Removed: **Settings_Nav-Normal_Good_Banner** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Settings_Nav-Normal_Good_Banner-33b46869301b.png)
 * Removed: **Settings_Nav-Normal_Bad_Banner** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481928211592-33b46869301b-6eb4211daf5a-linux/Settings_Nav-Normal_Bad_Banner-33b46869301b.png)
</summary>

**Comment by gabriel** [comment]
@keybase/react-hackers ready for review

File: shared/settings/dumb.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const updateEmailMap: DumbComponentMap<UpdateEmail> = {
     'Normal - No Email': {
       ...updateEmailBase,
       isVerified: false,
-      email: null,
+      email: '',
```
Comment (MarcoPolo) [inline shared/settings/dumb.js]: How is this different than null?

File: shared/settings/passphrase/index.desktop.js [hunk 2]
```diff
@@ -1,51 +1,94 @@
...
+    super(props)
+    this.state = {
+      passphrase: new HiddenString(''),
+      passphraseConfirm: new HiddenString(''),
+      showTyping: false,
+      canSave: false,
+    }
+  }
+
+  handlePassphraseChange (passphrase: string) {
```
Comment (MarcoPolo) [inline shared/settings/passphrase/index.desktop.js]: 🐑 change these to have an underscore e.g. `_handlePassphraseChange`

File: shared/settings/invites/index.desktop.js [hunk 3]
```diff
@@ -1,76 +1,114 @@
...
+      inviteEmail: props.inviteEmail,
+      inviteMessage: props.inviteMessage,
+      showMessageField: props.showMessageField,
+    }
+  }
+  componentWillUnmount () {
+    if (this.props.error) this.props.onClearError()
+  }
+
+  handleChangeEmail (inviteEmail: string) {
```
Comment (MarcoPolo) [inline shared/settings/invites/index.desktop.js]: 🐑 same underscore convention

**Comment by keybase-ci-visdiff** [comment]
The commits 1d6f52421bce...1078450d18c4 introduce visual changes on linux. <details><summary>:mag_right: 2 removed, 7 changed</summary>

 * Changed: **Update_Passphrase-Normal_Show_Typing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Normal_Show_Typing-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Normal_Show_Typing-1078450d18c4.png)  
   Update_Passphrase-Normal_Show_Typing comparison
 * Changed: **Update_Passphrase-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Normal-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Normal-1078450d18c4.png)  
   Update_Passphrase-Normal comparison
 * Changed: **Update_Passphrase-Error_Wrong_Passphrase** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Error_Wrong_Passphrase-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Error_Wrong_Passphrase-1078450d18c4.png)  
   Update_Passphrase-Error_Wrong_Passphrase comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Requirements** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Error_New_Passphrase_Requirements-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Error_New_Passphrase_Requirements-1078450d18c4.png)  
   Update_Passphrase-Error_New_Passphrase_Requirements comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Mismatch** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-1078450d18c4.png)  
   Update_Passphrase-Error_New_Passphrase_Mismatch comparison
 * Changed: **Settings_Nav-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Settings_Nav-Normal-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Settings_Nav-Normal-1078450d18c4.png)  
   Settings_Nav-Normal comparison
 * Changed: **Invites-Normal_Email_Error** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Invites-Normal_Email_Error-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Invites-Normal_Email_Error-1078450d18c4.png)  
   Invites-Normal_Email_Error comparison
 * Removed: **Settings_Nav-Normal_Good_Banner** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Settings_Nav-Normal_Good_Banner-1d6f52421bce.png)
 * Removed: **Settings_Nav-Normal_Bad_Banner** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482872469789-1d6f52421bce-1078450d18c4-linux/Settings_Nav-Normal_Bad_Banner-1d6f52421bce.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 1d6f52421bce...74d16bf269e1 introduce visual changes on linux. <details><summary>:mag_right: 2 removed, 7 changed</summary>

 * Changed: **Update_Passphrase-Normal_Show_Typing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Normal_Show_Typing-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Normal_Show_Typing-74d16bf269e1.png)  
   Update_Passphrase-Normal_Show_Typing comparison
 * Changed: **Update_Passphrase-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Normal-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Normal-74d16bf269e1.png)  
   Update_Passphrase-Normal comparison
 * Changed: **Update_Passphrase-Error_Wrong_Passphrase** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Error_Wrong_Passphrase-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Error_Wrong_Passphrase-74d16bf269e1.png)  
   Update_Passphrase-Error_Wrong_Passphrase comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Requirements** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Error_New_Passphrase_Requirements-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Error_New_Passphrase_Requirements-74d16bf269e1.png)  
   Update_Passphrase-Error_New_Passphrase_Requirements comparison
 * Changed: **Update_Passphrase-Error_New_Passphrase_Mismatch** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Update_Passphrase-Error_New_Passphrase_Mismatch-74d16bf269e1.png)  
   Update_Passphrase-Error_New_Passphrase_Mismatch comparison
 * Changed: **Settings_Nav-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Settings_Nav-Normal-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Settings_Nav-Normal-74d16bf269e1.png)  
   Settings_Nav-Normal comparison
 * Changed: **Invites-Normal_Email_Error** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Invites-Normal_Email_Error-1d6f52421bce.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Invites-Normal_Email_Error-74d16bf269e1.png)  
   Invites-Normal_Email_Error comparison
 * Removed: **Settings_Nav-Normal_Good_Banner** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Settings_Nav-Normal_Good_Banner-1d6f52421bce.png)
 * Removed: **Settings_Nav-Normal_Bad_Banner** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1482873008396-1d6f52421bce-74d16bf269e1-linux/Settings_Nav-Normal_Bad_Banner-1d6f52421bce.png)
</summary>

**Pull request merged**